test/util/compress_variant.cpp

```diff
@@ -50,19 +57,21 @@ void test_single_type_result()
     ));
 }
 
-
-int test_main(int, char* [])
+BOOST_AUTO_TEST_CASE(multiple_type)
 {
     test_variant_result<
         boost::variant<int, float, double>,
-        boost::mpl::vector<int, float, double>
+        boost::mpl::vector < int, float, double >
```

Comment:
Again my mistake, or rather an MSVC 2013 annoying feature that reformat things without user's consent...